### PR TITLE
LHC-2708 able to accurately refect required fields on ticket form now

### PR DESCRIPTION
**Pull request by rdai10**
### Description
https://issues.liferay.com/browse/LHC-2708?focusedCommentId=2204473&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2204473

### CODE REVIEW

File: theme/documentation/permissions.md [hunk 1]
```diff
@@ -8,19 +8,22 @@ Here is a breakdown of the user tags:
 
 - A limited support user, an employee, a customer, a partner, or a watcher, will have the **KB** tag.
 - Trial users will not have the **KB** tag.
-- The **KB** tag will allow the user to view the full content of Knowledge Base Articles, product landing pages, the mega menu, announcements, quick links on the homepage, and CTAs (calls to actions).
+- The **KB** tag will allow the user to view the full content of Knowledge Base Articles, all tabs of the product landing pages, the mega menu, announcements, quick links on the homepage, and CTAs (calls to actions). Note: Official Documentation i.e. the Documentation tab on product landing pages, is open to all users, the KB tag gives access to everything else.
```
Comment (jonmak08) [inline theme/documentation/permissions.md:11]: Should we wrap the `ie` in parentheses?

Note: Official Documentation (i.e. the Documentation tab on product landing pages), is open to all users; the KB tag gives access to everything else.

File: theme/documentation/permissions.md [hunk 2]
```diff
@@ -8,19 +8,22 @@ Here is a breakdown of the user tags:
...
 - The **Customer** or **Partner** role will be reflected as a tag on the user's organization; it follows the format of `osb_customer` or `osb_partner`.
 
 ## Watcher Tag
 
-- A user can be a **Watcher** for an organization without being a **Customer** or a **Partner**. Normally only **Customer** and **Partners** can open and view support tickets, this special role allows the user to view tickets associated with the organization that they belong to. However users with this role do not have the ability to create new tickets.
-- The **Watcher** role is reflected as a tag on the user's organization; it follows the format of `osb_${orgID}_watcher`.
\ No newline at end of file
+- A user can be a **Watcher** for an organization without being a **Customer** or a **Partner**. Normally only **Customer** and **Partners** can open and view support tickets, this special role allows the user to view tickets associated with the organization that they belong to. However users with this role do not have the ability to create new tickets or add comments to existing tickets.
```
Comment (jonmak08) [inline theme/documentation/permissions.md:23]: Need a comma after `However`.

However, users with this role do not have the ability to create new tickets or add comments to existing tickets.

File: theme/documentation/permissions.md [hunk 3]
```diff
@@ -8,19 +8,22 @@ Here is a breakdown of the user tags:
...
+- The **Watcher** role is reflected as a tag on the user's organization; it follows the format of `osb_${orgID}_watcher`.
+
+## Developer Services
+
+- The Developer Services link on the Help Center homepage is open only to users that are **Customers** or **Partners** (i.e. not a **Watcher**) on an organization with the Developer Services offering i.e. the organization will have the `developer_services` tag.
```
Comment (jonmak08) [inline theme/documentation/permissions.md:28]: Should we wrap the `ie` in parentheses?

The Developer Services link on the Help Center homepage is open only to users that are **Customers** or **Partners** (i.e. not a **Watcher**) on an organization with the Developer Services offering (i.e. the organization will have the `developer_services` tag).

File: theme/documentation/permissions.md [hunk 4]
```diff
@@ -8,19 +8,22 @@ Here is a breakdown of the user tags:
...
+- The knowledge base articles are also permissioned by segments. e.g. Developer Services - Customers or Developer Services - GS segment. The user needs to be added to one of these segments to be able to view the articles.
```
Comment (jonmak08) [inline theme/documentation/permissions.md:29]: Should we wrap the `eg` in parentheses?

The knowledge base articles are also permissioned by segments. (e.g. Developer Services - Customers or Developer Services - GS segment). The user needs to be added to one of these segments to be able to view the articles.

**Comment by rdai10** [comment]
Thanks @jonmak08 550e0195cc1ed8b5981f76e9c98f8747ad5143fe

**Comment by jonmak08** [comment]
LGTM.  Thanks @rdai10!

**Pull request merged**